{{cookiecutter.project_slug}}/backend/app/app/db/init_db.py

```diff
@@ -2,6 +2,8 @@
 from app.core import config
 from app.models.user import UserCreate
 
+from .base import *  # noqa
```

Comment:
the `# noqa` here is important, others wise the autoformat tools (lint.sh) would remove this line